src/backend/dev_helpers.rs

```diff
@@ -62,9 +62,28 @@ async fn clear_buckets() {
 #[update]
 fn replace_user_principal(principal: String, user_id: UserId) {
```

Comment:
I don't see this method being exposed anywhere?

Reply:
I think those are helper methods — see the file name.